samples/synchronization/src/main.c

```diff
@@ -10,10 +10,10 @@
 #include <zephyr/sys/printk.h>
 
 /*
- * The hello world demo has two threads that utilize semaphores and sleeping
+ * The synchronization demo has two threads that utilize semaphores and sleeping
  * to take turns printing a greeting message at a controlled rate. The demo
- * shows both the static and dynamic approaches for spawning a thread; a real
- * world application would likely use the static approach for both threads.
+ * shows only the dynamic approach for spawning a thread. Alternatively, 
+ * a thread can be declared at compile time by calling K_THREAD_DEFINE.
```

Comment:
thanks for catching the inaccuracy in the documentation! I think this would make for a great follow-up commit to actually make thread A or B a statically defined one, if possible? :) FWIW looks like the change dates back from 686b9fff5eb3391536185275fb7ac7794ef00dff.